packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -201,6 +223,18 @@ export function importAnalysisPlugin(config: ResolvedConfig): Plugin {
       server = _server
     },
 
+    resolveId(id) {
+      if (id === importAnalysisHelperId) {
+        return id
+      }
+    },
+
+    load(id) {
+      if (id === importAnalysisHelperId) {
+        return 'export ' + __vite__injectQuery.toString()
```

Comment:
I don't know if this is genius or if this language has way to much meta programming 😄